frontend/src/features/properties/list/interfaces.ts

```diff
@@ -69,7 +69,12 @@ export interface IPropertyFilter {
   quantity: number;
   address?: string;
   municipality?: string;
+  projectNumber?: string;
   classificationId?: number;
   statusId?: number;
-  agencies?: number[];
+  agencies?: number | number[];
+  minLandArea?: number;
+  maxLandArea?: number;
+  minLotArea?: number;
+  maxLotArea?: number;
```

Comment:
The server-side model accepts several "land area" parameters. I briefly tried both `minLandArea` and `minLotArea` and cannot get proper results. @Fosol maybe you can take a look tomorrow? 

Once this PR gets merged it will become very apparent that entering data in the Min/Max Area Size fields do not return results

Reply:
MinLand and MinLot are the same in the DB.  We don’t actually have a two different values.  Land = Lot.

Reply:
I suspect perhaps the building need to get a LotSize added, but am not certain as it wasn’t expressed this way in the original dataset.

Reply:
Yeah I don't know either what's going on. I will update the PR so it passes linting and once merged you can take a look at the back end